ui/admin/app/templates/scopes/scope/credential-stores/credential-store/worker-filter.hbs

```diff
@@ -0,0 +1,51 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+}}
+
+{{page-title (t 'resources.credential-store.titles.worker-filter')}}
+
+<Rose::Layout::Page as |page|>
+  <page.breadcrumbs>
+    <Breadcrumbs::Container />
+  </page.breadcrumbs>
+
+  <page.header>
+    <CredentialStores::CredentialStore::Header @model={{@model}} />
+  </page.header>
+
+  <page.navigation>
+    <CredentialStores::CredentialStore::Nav @model={{@model}} />
+  </page.navigation>
+
+  <page.actions>
+    <CredentialStores::CredentialStore::Actions
+      @model={{@model}}
+      @delete={{fn this.credentialStores.delete @model}}
+    />
+  </page.actions>
+
+  <page.body>
+    <Rose::Layout::Centered>
```

Comment:
I can remove the `Rose` component but it removes the padding-top so it will look like this now. Lmk what you suggest.
<img width="1044" alt="Screenshot 2024-11-19 at 12 49 02 PM" src="https://github.com/user-attachments/assets/118fb646-2ca9-40ea-8270-796d76640f9d">

Reply:
Fixed by adding a class to page.body and adding class specs to .scss 👍